src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -1668,6 +1668,22 @@ private DiagnosticInfo GetBadEventUsageDiagnosticInfo(EventSymbol eventSymbol)
                 new CSDiagnosticInfo(ErrorCode.ERR_BadEventUsageNoField, leastOverridden);
         }
 
+        internal static bool IsReadingThroughBackingFieldInConstructor(BoundPropertyAccess propertyAccess, Symbol fromMember)
```

Comment:
The "InConstructor" in the method name here looks important to me. A readonly auto property will always return true if we dropped "InConstructor" from the name, but that's not the behavior that callers are expecting.